### DISCUSSION TITLE
How to write back changes into the original resx files?

### DISCUSSION THREAD

**Discussion by joeysmart** [dotnet/ResXResourceManager#545] [Q&A] [answered]
Hi there!
I have imported some translations from an excel file, and now I am asking how I can write back this strings into the original resx files.
I am seeing a save icon in the left upper corner, but it is gray.

Any Ideas?

**Comment by tom-englert** [comment]
They are immediately saved

**Reply by joeysmart** [reply, marked as answer]
Thank you Tom for the fast response! In my case it was not written back for a while but now its there. Great tool, saves me a lot of time!
Best Regards 
Joe
